tools/tslint-rules/noUnscapedHtmlTagRule.js

```diff
@@ -0,0 +1,63 @@
+const ts = require('typescript');
+const utils = require('tsutils');
+const Lint = require('tslint');
+
+const ERROR_MESSAGE =
+  'An HTML tag may only appear in a JSDoc comment if it is escaped.' +
+  ' This prevents failures in document generation caused by a misinterpreted tag.';
+
+/**
+ * Rule that walks through all comments inside of the library and adds failures when it
+ * detects unescaped HTML tags inside of multi-line comments.
+ */
+class Rule extends Lint.Rules.AbstractRule {
+
+  apply(sourceFile) {
+    return this.applyWithWalker(new NoUnescapedHtmlTagWalker(sourceFile, this.getOptions()));
+  }
+}
+
+class NoUnescapedHtmlTagWalker extends Lint.RuleWalker {
+
+  visitSourceFile(sourceFile) {
+    utils.forEachComment(sourceFile, (fullText, commentRange) => {
+
+      let isEscapedHtmlTag = true;
+
+      const numberOfBackticks = fullText.split('`').length - 1;
+      // if there are no backticks no check is necessary
+      if (numberOfBackticks > 0) {
+        // if there are backticks there should be an even number of them
+        if (!!(numberOfBackticks % 2)) {
```

Comment:
Should be fine to just use `if(numberOfBackticks % 2)` since you only care if it's zero or non-zero